LICENSE

```diff
@@ -1,5 +1,7 @@
 MIT License
 
+Copyright (c) Leon Bichmann
```

Comment:
Normally add everyone who contributed significantly to the code (e.g. Lukas, ...)

Reply:
added Lukas and you